Update ligo.osgstorage.org.conf - default port

biozit opened the pull request:
caches are using ports 8444 and 8443, setting the default port (8443)

DrDaveD reviewed (changes requested):
No, we discussed putting in both ports for a transition period, and increasing the number of CVMFS_EXTERNAL_MAX_SERVERS during that period so the number of real servers that a cvmfs client will get won't be decreased below 4.

biozit commented:
Hi Dave,
   
    I changed the config on caches, now they can provide auth files using ports 8444 and 8443. If we changed the config, the cache still work for both ports 8444 and 8443. After the config is applied, I will remove port 8444 from the PODs.

DrDaveD reviewed (approved):
Oh!  That's a better way to do it than what I was thinking.  I didn't know that was an option. 

I'll need you to also submit corresponding changes to the egi and master branches.

DrDaveD commented:
Oh I see you already did master in #123, so we only need one for egi.

biozit commented:
Hi Dave, I created the other PR.

DrDaveD commented:
@jthiltges This is waiting on you to merge and distribute as well.
